examples/ipfs-from-typed-js/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "ipfs-from-ts",
```

Comment:
CI is failing because the name `ipfs-from-ts` is used for both new examples.

Suggest renaming this one (and the directory it's in) to `types-use-ipfs-from-typed-js`